docs/cloud-native-security/d4c-policy-guide.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[d4c-policy-guide]]
+= Guide to container workload protection policies
+
+To unlock the full functionality of the Defend for Containers (D4C integration), you'll need to understand its policy syntax. This will enable you to construct policies that precisely allow expected container behaviors and prevent unexpected behaviors – thereby hardening the security posture of your container workloads.
+
+Policies consist of _Selectors_ and _Responses_. A given policy must contain at least one selector and one response. Currently, the system supports two types of selectors and responses: `file` and `process`.
+Selectors define which system operations to match, and can include multiple conditions (grouped using a logical `AND`) to precisely select events. Responses define which actions to take when a system operation matches the conditions specified in the associated selector.
+
+The default policies described on this page serve as examples for understanding D4C policies. Below that description, you will find an exhaustive glossary of selector conditions, and of response fields and actions.
+
+[[d4c-default-policies]]
+[discrete]
+== Default policies:
+By default, the D4C integration includes two policies. They are designed to implement core container workload protection capabilities:
```

Comment:
From the screenshot and description below, it actually doesn't look like there are two policies here. It looks like one integration policy (named `cloud_defend-1` in the image), which contains two _categories_ of settings (`allProcesses` and `executableChanges`).

I don't know if this is semantic hairsplitting or actually significant, but I kinda think it'd be confusing to someone familiar with how Elastic's other integrations and integration policies work. If we say "there are two policies," they'd probably be expecting two entirely separate policy types, or at least two separate policies of the same type, not a single policy.

Reply:
Thank you for this. Great example of a second pair of eyes coming in clutch.